libs/ledger-live-common/src/families/tezos/bridge/js.ts

```diff
@@ -270,8 +274,10 @@ const prepareTransaction = async (
 
     tx.storageLimit = new BigNumber(estimate.storageLimit);
     tx.estimatedFees = tx.fees;
+
     if (!tezosResources.revealed) {
       tx.estimatedFees = tx.estimatedFees.plus(DEFAULT_FEE.REVEAL);
+      tx.fees = tx.fees.plus(DEFAULT_FEE.REVEAL);
```

Comment:
Did you try with just this update and without lib update ?

It looks like the PR does this and lib update

Suggestion but maybe we could do :

```
if(!tezosResources.revealed) {
 tx.fees = tx.fees.plus(DEFAULT_FEE.REVEAL);
 }
 tx.estimatedFees = tx.fees
```

Instead of doing it twice

Reply:
This is still needed to show the correct fees to the user in the UI

Reply:
Ok !